docs/user.md

```diff
@@ -0,0 +1,193 @@
+2. Label node with `node-role.kubernetes.io/tests=""` (certification related pods will schedule to dedicated node)
+3. Taint node with `node-role.kubernetes.io/tests="":NoSchedule` (prevent other pods from running on dedicated node)
+
+*NOTE: certification pods will automatically have node-selectors and taint tolerations if you use the `--dedicated` flag.*
+
+There are two options to accomplish this type of setup...
+
+##### Option A: Command Line 
+
+```shell
+oc label node <node_name> node-role.kubernetes.io/tests=""
+oc taint node <node_name> node-role.kubernetes.io/tests="":NoSchedule
+```
+
+##### Option B: Machine Set 
+
+If you have support for OpenShift's Machine API then you can create a new `MachineSet` to configure the labels and taints. See [OpenShift documentation](https://docs.openshift.com/container-platform/latest/machine_management/creating-infrastructure-machinesets.html#binding-infra-node-workloads-using-taints-tolerations_creating-infrastructure-machinesets) on how to configure a new `MachineSet`. Note that at time of certification testing, OpenShift's product documentation may not mention your infrastructure provider yet! 
+
+Here is a `MachineSet` YAML snippet on how to configure the label and taint as well:
+
+```yaml
+      metadata:
+        labels:
+          openshift-tests: "true"
+          node-role.kubernetes.io/tests: ""
+      taints:
+        - key: node-role.kubernetes.io/tests
+          effect: NoSchedule
+```
+
+
+### Privilege Requirements
+
+A user with [cluster administrator privilege](https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#creating-cluster-admin_using-rbac) must be used with the provider certification tool. You also use the default kubeadmin user if you wish. 
+
+
+## Install
+
+
+### Prebuilt Binary
+
+The provider certification tool is shipped as a single executable binary which can be downloaded from:
+
+[https://github.com/redhat-openshift-ecosystem/provider-certification-tool](https://github.com/redhat-openshift-ecosystem/provider-certification-tool)
+
+The provider certification tool can be used from any system with network access to the OpenShift cluster under test. 
+
+
+### Build from Source
+
+See the development guide for more information.
+*TODO - link*
+
+
+## Usage
+
+
+### Run provider certification tests
+
+```sh
+openshift-provider-cert run 
+openshift-provider-cert run -w # Keep watch open until completion
+```
+
+
+### Check status
+
+```sh
+openshift-provider-cert status 
+openshift-provider-cert status -w # Keep watch open until completion
+```
+
+
+### Retrieve the results
+
+The certification results must be retrieved from the OpenShift cluster under test using:
+
+```sh
+openshift-provider-cert retrieve
+openshift-provider-cert retrieve ./destination-dir/
+```
+
+You can see a summarized view of the results using:
+
+```sh
+openshift-provider-cert results retrieved-archive.tar.gz
+```
+
+
```

Comment:
I believe we need to define the workflow to "send" the results, and also to provide feedback.

Currently, the results are more than 50MiB on the tarball with gzip. I am wondering if we need to provide an `upload` subcommand on CLI, that will perform tasks like:

- untar the current gzipped file
- apply any change on the result file (inserting metadata about the provider/partner, etc)
- tar the results again using the XZ compression, it will result in more than 10x the actual size using gzip
- provide feedback on the new file and the steps to send to us (which was discussed that will be, initially, by email). TBD

Reply:
I think we could do that as part of the retrieve command so its transparent to the user. What do you think?

Reply:
Having an ending message on the `retrieve` will be a good idea.

Reply:
Do you think just a message or should we add the functionality you mentioned? (Or should we file a card that's "nice to have"?)

Reply:
I believe just printing a message to contact the partner manager, as we commented on in yesterday's meeting.

Reply:
Resolving this conversation and moving to https://issues.redhat.com/browse/SPLAT-638